static/app/utils/useUserTeams.tsx

```diff
@@ -54,8 +54,10 @@ export function useUserTeams(): UseTeamsResult {
 
   const isSuperuser = isActiveSuperuser();
   const teams = useMemo<Team[]>(() => {
-    return isSuperuser ? storeState.teams : storeState.teams.filter(t => t.isMember);
-  }, [storeState.teams, isSuperuser]);
+    return isSuperuser || loadAllTeams
```

Comment:
Related to my comment above, but we would do something like
```suggestion
    return isSuperuser || organization.access.includes('org:admin')
```